cmd/main.go

```diff
@@ -120,7 +122,13 @@ func main() {
 		log.Fatal("failed to auto migrate, ", err)
 	}
 
-	authHandler := auth.NewAuthHandler(Options.Auth, log.WithField("pkg", "auth"))
+	ocmClient, err := ocm.NewClient(Options.OCMConfig)
```

Comment:
Perhaps we should spawn an `ocm-client` only when `Options.Auth.EnableAuth` is set to true.
You can pass a `nil` ocmClient to `NewAuthHandler()`, same as you do in unit tests.

Reply:
sure, changed new client creation to be invoked only when auth is enabled.